src/api/public/apidocs-new/paths/staging_staging_workflow_project_staging_projects_staging_project_name_copy_project_name.yaml

```diff
@@ -0,0 +1,60 @@
+post:
+  summary: Copy a staging project
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/staging_workflow_project.yaml'
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: path
+      name: staging_project_copy_name
+      schema:
+        type: string
+      required: true
+      description: The name of the staging project's copy
+      example: "openSUSE:Factory:Staging:B"
+  responses:
+    '200':
+      description: Get the state of a staging project.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/staging/project.yaml'
+          example:
+            name: 'home:Iggy:Staging:A'
+            staged_requests:
+              count: 0
+            untracked_requests:
+              count: 0
+            obsolete_requests:
+              count: 0
+            missing_reviews:
+              count: 0
+            building_repositories:
+              count: 0
+            broken_packages:
+              count: 0
+            checks:
+              count: 0
+            missing_checks:
+              count: 0
+            history:
+              count: 0
```

Comment:
The response 200 is a common `OK` response:

```
<status code="ok">
  <summary>Ok</summary>
</status>
```

Reply:
True